tests/test_gestalt.py

```diff
@@ -7,6 +7,49 @@
 import hvac
 
 
+# Testing member function
+def test_combined_into():
+    g = gestalt.Gestalt()
+    dictionary = {}
+    dictionary_ = {} 
```

Comment:
`dictionary_` feels off. perhaps name these `combined1` and `combined2`?

Reply:
updated